gomint-server/src/main/java/io/gomint/server/inventory/item/ItemPotion.java

```diff
@@ -21,7 +25,125 @@ public ItemType getType() {
 
     @Override
     public void onConsume( EntityPlayer player ) {
-        // Apply effects of the potion
+        switch ( this.getData() ) {
+            case 0:
```

Comment:
Please use either a enum or private constants with speaking names of the potion.